docs/IE_DG/Extensibility_DG/Custom_ONNX_Ops.md

```diff
@@ -50,9 +50,9 @@ The example below demonstrates how to unregister an operator from the destructor
 
 ## Requirements for Building with CMake
 
-A program that uses the `register_operator` functionality requires `ngraph` and `onnx_importer` libraries in addition to the Inference Engine.
-The `onnx_importer` is a component of the `ngraph` package , so `find_package(ngraph REQUIRED COMPONENTS onnx_importer)` can find both.
-The `ngraph` package exposes two variables, `${NGRAPH_LIBRARIES}` and `${ONNX_IMPORTER_LIBRARIES}`, which reference the `ngraph` and `onnx_importer` libraries.
+A program that uses the `register_operator` functionality requires `ngraph` and `onnx_ngraph_frontend` libraries in addition to the Inference Engine.
+The `onnx_ngraph_frontend` is a component of the `ngraph` package , so `find_package(ngraph REQUIRED COMPONENTS onnx_ngraph_frontend)` can find both.
+The `ngraph` package exposes two variables, `${NGRAPH_LIBRARIES}` and `${ONNX_FRONTEND_LIBRARIES}`, which reference the `ngraph` and `onnx_ngraph_frontend` libraries.
```

Comment:
can we remove `ONNX_FRONTEND_LIBRARIES` variables and work with importer targets directly?

Also, I believe we need to add suffix `ngraph::` to all importer targets